.github/workflows/build-deploy.yml

```diff
@@ -0,0 +1,23 @@
+name: Build and Deploy Debezium Website
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  build-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build
+        run: docker run --rm --privileged=true --user=$UID:$UID -i -p 4242:4242 -e LC_ALL=C.UTF-8 -e LANG=C.UTF-8 -v $(pwd):/site -v ~/github/debezium:/debezium debezium/awestruct bash
```

Comment:
Similar to my comments on the `build.yml`, the same holds true here.  If we intend to use the docker container to build and deploy to GitHub, we'd need to call this container in the exact same way we do for the `build.yml` with a few minor exceptions:

1. Do not pass `TRAVIS_PULL_REQUEST=1` but instead `TRAVIS_PULL_REQUEST=0`.
2. The command you would run remains the same, `rake clean documentation travis`.

This would trigger the building of documentation & the website as well as deployment to GH pages.